src/System.Management.Automation/DscSupport/ConvertCimMofToJsonCommand.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Management.Automation;
+using System.Text;
+
+namespace Microsoft.PowerShell.DesiredStateConfiguration.Internal
+{
+    /// <summary>
+    /// Convert-CimMofToJson cmdlet implementation
+    /// </summary>
+    [Cmdlet(VerbsData.Convert, "CimMofToJson")]
```

Comment:
Isn't this DSC specific?  
```suggestion
    [Cmdlet(VerbsData.Convert, "DscMofToJson")]
```

Reply:
The command verb and noun really should be changed to adhere to PS verb naming guidelines. `Convert` indicates bi-directional conversion and this is only one way. So resolve that the command name would be updated to `ConvertTo-DscJson` I have more info here in my other comment https://github.com/PowerShell/PowerShell/pull/13399#issuecomment-672124381

Reply:
Cmdlet was renamed to `ConvertTo-DscJsonSchema` and moved to `PSDesiredStateConfiguration` module.